src/app/comment-period/submit-comment-modal/submit-comment-modal.ts

```diff
@@ -73,20 +72,25 @@ export class SubmitCommentModalComponent implements OnInit {
   }
 
   validateFields(form) {
+    this.clearErrors();
+    let validAuthor = true;
```

Comment:
Any reason to have 4 separate variables, rather than just 1?
Also we should type these as booleans.